reference-implementation/lib/writable-stream.js

```diff
@@ -158,7 +162,6 @@ function WritableStreamError(stream, e) {
   }
 
   stream._state = 'errored';
```

Comment:
Can we keep this line together with assigning storedError? It seems bad for them to get out of sync.